.github/workflows/preview.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Any Commit
+on:
+    pull_request:
+        types: [opened, synchronize, reopened]
+    push:
+        branches:
+            - master
+
+jobs:
+    build:
+        runs-on: ubuntu-22.04
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Setup Node
+              uses: actions/setup-node@v4
+              with:
+                  node-version: '20.18.1'
+                  cache: 'yarn'
+
+            - name: Install dependencies
+              run: yarn install --frozen-lockfile
+
+            - name: Publish preview
+              run: npx pkg-pr-new publish './packages/@lwc/*' './packages/lwc' --template './playground'
+              env:
+                  GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Does the token require any particular perms?

Reply:
- https://github.com/stackblitz-labs/pkg.pr.new/issues/305

Seems pretty basic to me. I think it even runs without the token, it just won't provide the comment, and won't update when new commits are pushed.